api/src/main/java/jakarta/xml/bind/annotation/XmlElementRef.java

```diff
@@ -144,7 +144,7 @@
  * that doesn't have {@link XmlRootElement}. But they can't show up in an
  * XML instance (because they don't have XML element names).
  *
- * <p><b>Example 2: XML Schema Susbstitution group support</b>
+ * <p><b>Example 2: XML Schema substitution group support</b>
```

Comment:
first letter should be kept in upper case

Reply:
Fixed. However, At first commit I copied same format than line 37. Check it out and say to me if it is necessary to fix too.

Regards,
Antonio.

Reply:
the best would be to bring in consistency accross all usages ;-)

Reply:
Fixed in XmlElementDecl too!